server/src/services/person.service.ts

```diff
@@ -130,6 +146,34 @@ export class PersonService {
     return result;
   }
 
+  async unassignFaces(auth: AuthDto, dto: AssetFaceUpdateDto): Promise<BulkIdResponseDto[]> {
+    const changeFeaturePhoto: string[] = [];
+    const results: BulkIdResponseDto[] = [];
```

Comment:
My OCD really likes that these two lines have the exact same length lmao

Reply:
They just line up so perfectly!

Reply:
What's the problem with it ?

Reply:
Oh nothing, I was just happy about that lol

Reply:
That was a positive comment :D

Reply:
Oh damn, I just saw it. Satisfying :smile: